WordPress/Classes/Services/PostRepository.swift

```diff
@@ -63,6 +58,58 @@ final class PostRepository {
         }
     }
 
+    /// Uploads the changes to the given post to the server and deletes the
+    /// uploaded local revision afterward. If the post has an associated
+    /// remote ID, creates a new post.
```

Comment:
tiny nit - I think there's a typo

```suggestion
    /// uploaded local revision afterward. If the post doesn't have an associated
    /// remote ID, creates a new post.
```